automation_testing/tests/Components/form/validators.spec.ts

```diff
@@ -64,8 +64,7 @@ test.describe.parallel("Components - Form - Validators", () => {
 		await validatorPage.title.click();
 		const error = validatorPage.error;
 		expect(await error.textContent()).toBe(validatorData.emailError);
-		await validatorPage.emailField.selectText();
-		await validatorPage.clearAllValuesFromField();
+		await validatorPage.clearAllValuesFromField(validatorPage.emailField);
```

Comment:
I can't add the comment where it belongs but I got this error when running the tests
![image](https://user-images.githubusercontent.com/23409284/207467339-25bb47f0-f348-41a1-85f8-091622a5ef2f.png)

Reply:
I wasn't able to replicate the error but I know what is causing it. I implemented a fix, please run the test again and tell me if it passes.